src/components/Auth/Login/Login.react.js

```diff
@@ -55,6 +55,7 @@ class Login extends Component {
       showCaptchaErrorMessage: false,
       attempts: storageService.get('loginAttempts', 'session') || 0,
       captchaResponse: '',
+      wrongCredentialsMessage: '',
```

Comment:
Rename it to `message`

Reply:
Renamed it to `errorMessage` as `message` variable is already declared.